exporter/observer/observer.go

```diff
@@ -126,15 +126,15 @@ func Record(event Event) core.EventID {
 	}
 
 	observers, _ := observers.Load().(observersMap)
-	for observer, _ := range observers {
+	for observer := range observers {
 		observer.Observe(event)
 	}
 	return event.Sequence
 }
 
 func Foreach(f func(Observer)) {
 	observers, _ := observers.Load().(observersMap)
```

Comment:
nit: Could also drop the `,_`

Reply:
you cannot remove `, _` for classic variable assignments, only in `for` loops, as i have done here.

Reply:
You can elide the `,_` in a type assertion, but it becomes a panic, which I had forgotten about: https://golang.org/ref/spec#TypeAssertion